query-node/mappings/src/content/channel.ts

```diff
@@ -151,129 +133,39 @@ export async function content_ChannelAssetsRemoved({ store, event }: EventContex
-  })
-  integrateMeta(channelCategory, metadata, ['name'])
-
-  // save channel
-  await store.save<ChannelCategory>(channelCategory)
+export async function content_ChannelDeleted({ store, event }: EventContext & StoreContext): Promise<void> {
+  const [, channelId] = new Content.ChannelDeletedEvent(event).params
 
-  // emit log event
-  logger.info('Channel category has been created', { id: channelCategory.id })
+  await store.remove<Channel>(new Channel({ id: channelId.toString() }))
 }
 
-export async function content_ChannelCategoryUpdated({ store, event }: EventContext & StoreContext): Promise<void> {
-  // read event data
-  const [, channelCategoryId, channelCategoryUpdateParameters] = new Content.ChannelCategoryUpdatedEvent(event).params
+export async function content_ChannelOwnerRemarked(ctx: EventContext & StoreContext): Promise<void> {
+  const { event, store } = ctx
+  const [channelId, message] = new Content.ChannelOwnerRemarkedEvent(ctx.event).params
 
-  // load channel category
-  const channelCategory = await store.get(ChannelCategory, {
-    where: {
-      id: channelCategoryId.toString(),
-    },
+  // load channel
+  const channel = await store.get(Channel, {
+    where: { id: channelId.toString() },
+    relations: ['ownerMember', 'ownerCuratorGroup'],
   })
 
   // ensure channel exists
-  if (!channelCategory) {
-    return inconsistentState('Non-existing channel category update requested', channelCategoryId)
+  if (!channel) {
+    return inconsistentState('Owner Remarked for Non-existing channel', channelId)
   }
 
-  // read metadata
-  const newMeta = deserializeMetadata(ChannelCategoryMetadata, channelCategoryUpdateParameters.new_meta) || {}
-  integrateMeta(channelCategory, newMeta, ['name'])
-
-  // set last update time
-  channelCategory.updatedAt = new Date(event.blockTimestamp)
-
-  // save channel category
-  await store.save<ChannelCategory>(channelCategory)
-
-  // emit log event
-  logger.info('Channel category has been updated', { id: channelCategory.id })
-}
-
-export async function content_ChannelCategoryDeleted({ store, event }: EventContext & StoreContext): Promise<void> {
-  // read event data
-  const [, channelCategoryId] = new Content.ChannelCategoryDeletedEvent(event).params
-
-  // load channel category
-  const channelCategory = await store.get(ChannelCategory, {
-    where: {
-      id: channelCategoryId.toString(),
-    },
-  })
-
-  // ensure channel category exists
-  if (!channelCategory) {
-    return inconsistentState('Non-existing channel category deletion requested', channelCategoryId)
+  const getcontentActor = (ownerMember?: Membership, ownerCuratorGroup?: CuratorGroup) => {
+    if (ownerMember) {
+      const actor = new ContentActorMember()
+      actor.memberId = ownerMember.id
+      return actor
+    } else if (ownerCuratorGroup) {
+      const actor = new ContentActorCurator()
+      actor.curatorId = ownerCuratorGroup.id
+      return actor
+    }
   }
```

Comment:
There's a small typo in the function name and we can also prevent unexpected state that allows function to return `undefined`.

```suggestion
  const getContentActor = (ownerMember?: Membership, ownerCuratorGroup?: CuratorGroup) => {
    if (ownerMember) {
      const actor = new ContentActorMember()
      actor.memberId = ownerMember.id
      return actor
    }

    if (ownerCuratorGroup) {
      const actor = new ContentActorCurator()
      actor.curatorId = ownerCuratorGroup.id
      return actor
    }

    return inconsistentState('Unknown content actor', { ownerMember, ownerCuratorGroup })
  }
```